packages/knip/src/plugins/index.ts

```diff
@@ -54,3 +54,4 @@ export { default as vue } from './vue/index.js';
 export { default as webpack } from './webpack/index.js';
 export { default as wireit } from './wireit/index.js';
 export { default as yorkie } from './yorkie/index.js';
+export { default as wrangler } from './wrangler/index.js';
```

Comment:
Please move it up so it's ordered alphabetically

Reply:
Fixed in the latest commit